modules/multibid/index.ts

```diff
@@ -14,12 +14,44 @@ import {addBidderRequests} from '../../src/auction.js';
 import {getHighestCpmBidsFromBidPool, sortByDealAndPriceBucketOrCpm} from '../../src/targeting.js';
 import {PBS, registerOrtbProcessor, REQUEST} from '../../src/pbjsORTB.js';
 import {timedBidResponseHook} from '../../src/utils/perfMetrics.js';
+import type {BidderCode} from "../../src/types/common.d.ts";
 
 const MODULE_NAME = 'multibid';
 let hasMultibid = false;
 let multiConfig = {};
 let multibidUnits = {};
 
+type MultiBidConfig = ({
+    /**
+     * A bidder code.
+     */
+    bidder: BidderCode;
+    bidders: undefined
```

Comment:
```suggestion
    bidders: never
```

I think this would be an ever better representation? https://www.typescriptlang.org/docs/handbook/2/functions.html#never

Reply:
Judging by the other stuff you wrote, my suggestion is probably bullshit. Feel free to correct me 🙈